example/package.json

```diff
@@ -12,8 +12,8 @@
     "express": "^4.12.3",
     "logfmt": "^1.2.0",
     "node-jsx": "^0.13.3",
-    "react": "^0.14.0",
-    "react-dom": "^0.14.0",
+    "react": "^0.15.0",
+    "react-dom": "^0.15.0",
```

Comment:
Hey @mathieulesniak I'm happy to merge that but could you please fix the version number here too?